umh.docs.umh.app/content/en/docs/getstarted/movingtoproduction.md

```diff
@@ -1,22 +1,16 @@
-+++
-title = "5. Moving to Production"
-menuTitle = "5. Moving to Production"
-description = "Move the United Manufacturing Hub to production."
-weight = 5000
-+++
-
-  The next big step is to use the UMH on a virtual machine or an edge device in
-  your production and connect your production assets. However, we understand that
-  you might want to understand a little bit more about the United Manufacturing
-  Hub first. So, you can either read more about our product, deep-dive into your 
-  local installation, or continue with the deployment in production.
-  You can use the guide below to get started with the UMH in production 
-  and connect a few machines to a single instance.
+---
+title: 5. Moving to Production
+menuTitle: 5. Moving to Production
+description: Move the United Manufacturing Hub to production.
+weight: 5000
+---
+
+Discover the power of the United Manufacturing Hub (UMH) — where efficiency meets insight. Whether you're eager to dive into local/production deployment or just wish to explore more about our product, the choice is yours.
 
 ## Check out our community
 
-  We are quite active on GitHub and Discord. Feel free to join, introduce
-  yourself and share your best-practices and experiences.
+We are quite active on GitHub and Discord. Feel free to join, introduce
```

Comment:
You could also link here: https://www.umh.app/community (so we dont have separate links to discord, etc. everywhere and just one community page)